js_modules/dagit/packages/app/src/NUX/CommunityNux.tsx

```diff
@@ -212,3 +220,20 @@ const RecaptchaIFrame: React.FC<{
 };
 
 const IFRAME_SRC = '//dagster.io/dagit_iframes/community_nux';
+
+const SET_NUX_SEEN_MUTATION = gql`
+  mutation SetNuxSeen {
+    setNuxSeen
+  }
+`;
+
+const GET_SHOULD_SHOW_NUX_QUERY = gql`
+  query ShouldShowNux {
+    shouldShowNux
+  }
+`;
+
+function isLocalhost() {
+  const origin = window.location.origin;
+  return origin.startsWith('http://127.0.0.1') || origin.startsWith('http://localhost');
```

Comment:
i assume people using https here is not a thing

Reply:
<img width="644" alt="Screen Shot 2023-01-17 at 6 26 05 PM" src="https://user-images.githubusercontent.com/2286579/213034392-3bde1bf2-741c-44a7-8d11-b61399096c99.png">

Yea